lib/core/context.c

```diff
@@ -173,7 +173,7 @@ void h2o_get_timestamp(h2o_context_t *ctx, h2o_mem_pool_t *pool, h2o_timestamp_t
             if (ctx->_timestamp_cache.value != NULL)
                 h2o_mem_release_shared(ctx->_timestamp_cache.value);
             ctx->_timestamp_cache.value = h2o_mem_alloc_shared(NULL, sizeof(h2o_timestamp_string_t), NULL);
-            gmtime_r(&ctx->_timestamp_cache.tv_at.tv_sec, &gmt);
+            gmtime_r((time_t *)&ctx->_timestamp_cache.tv_at.tv_sec, &gmt);
```

Comment:
I will remove the cast.
I need to look at the differences between windows and unix headers closely here